ml/RL/env.py

```diff
@@ -185,14 +256,124 @@ def reset(self):
                                              heading_angle=self.state[2],
                                              steering_angle=self.state[4]
                                              )
+        self.track = self._create_track()
 
         return self.init_observation
 
     def render(self, mode='human'):
```

Comment:
probably adding some comments to the function could be useful, so that we can improve the rendering after Christmas